README.rst

```diff
@@ -540,6 +557,20 @@ Full List of Settings
 ``EVENT_CERT_DOMAINS``, ``BOUNCE_CERT_DOMAINS``
   Optional. Default is 'amazonaws.com' and 'amazon.com'.
 
+``AWS_SES_ENABLE_BOUNCE_BLACKLIST``
+  If set to ``True`` (default ``False``) email addresses that triggered an
+  irrecoverable bounce (status in the ``5xx`` range) will be added to the
+  blacklist.
+
+``AWS_SES_ENABLE_COMPLAINT_BLACKLIST``
+  If set to ``True`` (default ``False``) email addresses that triggered a complaint
+  will be added to the blacklist.
+
+``AWS_SES_USE_BLACKLIST``
+  If set to ``True`` (default ``False``), calls to the ``send_mail()`` method will
+  cause the recipients to be filtered using the blacklist. Any recipient that
+  exists in the blacklist will be removed from the email.
```

Comment:
Do you think it could be confusing that a user must enable the `AWS_SES_USE_BLACKLIST` setting to actually prevent sending emails to those addresses?

I agree it can be nice to have a flag to disable the filtering, so that you can first accumulate emails and see who would be blocked. Maybe we would rename the first two settings to `AWS_SES_ADD_BOUNCE_TO_BLACKLIST` (replacing ENABLE with ADD). What do you think?

Reply:
Hmmm 🤔… yes… It might be confusing to have to double-enable the feature (one time for the `ADD_*` and a second time for the actual filtering). Maybe the `AWS_SES_USE_BLACKLIST` should be set to `True` by default (while keeping the other two to `False` by default)?

About the other two, yes, replacing `ENABLE` with `ADD` sounds good!